package/network/utils/umbim/files/lib/netifd/proto/mbim.sh

```diff
@@ -150,6 +153,7 @@ _proto_mbim_setup() {
 	tid=$((tid + 1))
 
 	pdptype=$(echo "$pdptype" | awk '{print tolower($0)}')
+	[ "$ipv6" = 0 ] && pdptype="ipv4"
```

Comment:
Now `req_pdptype` will never be empty if IPv6 is not supported

Reply:
This is expected, with IPv6 disabled, we can force IPv4 PDN connection in the next step anyway.